packages/@aws-cdk/integ-runner/README.md

```diff
@@ -97,6 +97,17 @@ If not, changes cannot be compared across systems.
   - `go`:
     - File RegExp: `^integ_.*\.go$`
     - App run command: `go run {filePath}`
+- `--list` (default=`false`)
+  List tests instead of running them.
+- `--inspect-failures` (default=`false`)
+  Keep generated snapshots when differences exist in snapshot comparisons.
+- `--dry-run` (default=`false`)
```

Comment:
We dont want to document this one :)

Reply:
@kaizencc 
I see. Deleted the `dry-run` option.